modules/cluster-logging-collector-chunk.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * cluster-logging-collector.adoc
+
+[id="cluster-logging-collector-chunk_{context}"]
+= Configuring Buffer Chunk Limiting for Fluentd
+
+If the Fluentd logger is unable to keep up with a high number of logs, 
+Fluentd peforms file buffering to reduce memory usage and prevent data loss.
+
+Fluentd file buffering stores records in _chunks_. Chunks are stored in _buffers_.
+
+You can tune file buffering in your cluster by editing environment variables
+in the `fluentd` daemonset:
+
+* *BUFFER_SIZE_LIMIT*. This parameter determines the maximum size of each chunk 
+file before Fluentd creates a new chunk. The default is `8M`. 
+This parameter sets the Fluentd `buffer_chunk_limit` variable.
++
+A high BUFFER_SIZE_LIMIT can collect more records per chunk file. However, 
+bigger records need more time to be sent to the logstore. 
+
+* *FILE_BUFFER_LIMIT*. This parameter determines the file buffer size per output.
+The default is `256Mi`.
++
+The permanent volume size must be larger than`FILE_BUFFER_LIMIT` multiplied by the
+number of outputs.
++
+A high FILE_BUFFER_LIMIT could translate to a higher BUFFER_QUEUE_LIMIT based the number of outputs. 
+However, if the node's space is under pressure Fluentd can fail.
++
+By default, the `number_of_outputs` is `1` if all the logs are sent to a single resource, and is incremented by `1` for each 
+additional resource. You might have multiple outputs if you use the Log Forwarding API, the Fluentd protocols, or 
+syslog protocol to forward logs to external locations.
++
```

Comment:
@periklis Is there a way to view the number of outputs?

Reply:
Actually not. The variable is not exported to env anymore.